COMET.Web.Common/Extensions/SessionServiceExtensions.cs

```diff
@@ -66,7 +66,7 @@ public static async Task<Result> AddParameter(this ISessionService sessionServic
 
             if (sessionService.Session == null)
             {
-                throw new ArgumentNullException(nameof(sessionService.Session), "The session may not be null");
+                throw new ArgumentNullException(nameof(sessionService), "The session contained by the given session service may not be null");

```

Comment:
i think you are giving the impression the sessionService is ull, that is not the case, the Session property is null

Reply:
Indeed, just updated this exception from "ArgumentNullException" to "InvalidOperationException" :)